cluster_loader/cluster_loader.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+import os
+import argparse
+import configparser
+import logging
+
+import trigger_cluster_loader
+
+logger = logging.getLogger("snafu")
+
+class cluster_loader_wrapper():
+
+    def __init__(self, parser):
+        #collect arguments
+
+        # it is assumed that the parser was created using argparse and already knows
+        # about the --tool option
+        parser.add_argument(
+            '-s', '--samples',
+            type=int,
+            help='number of times to run benchmark, defaults to 1',
+            default=1)
+        parser.add_argument(
+            '-d', '--dir',
+            help='output parent directory, defaults to current directory',
+            default=os.path.dirname(os.getcwd()))
+        parser.add_argument(
+            '-p', '--path-binary',
+            help='absolute path to openshift-tests binary',
+            default='/root/go/src/github.com/openshift/origin/_output/local/bin/linux/amd64/openshift-tests')
+        self.args = parser.parse_args()
+
+        self.server = ""
```

Comment:
What will this be used for? I don't see it used anywhere?

Reply:
Me either, will remove this, i used fs-drift/smallfile as the template to add this, so it's a potential leftover :)